### DISCUSSION TITLE
Default "Craft News" widget

### DISCUSSION THREAD

**Discussion by chrisemory** [craftcms/cms#8769] [Ideas]
### Description
I accidentally deleted my "Craft News" widget. I could not find the URL in the Control Panel or on your website to reinstall the widget. It would be great if "Craft News" was a standard option under "New widget"--> Feed

### Steps to reproduce

1. Delete Craft News widget
2. Try to find URL to reinstall Craft News widget
3. Had to ask another user for URL

### Additional info

- Craft version: 3.3.20.1 (Pro)
- PHP version: 7.2.24
- OS version: Linux 4.15.0-1057-aws
- Database driver & version: MySQL 5.7.28
- Image driver & version: Imagick 3.4.3RC2 (ImageMagick 6.9.7-4)
- Plugins & versions: 
    - CKEditor 1.0.0-beta.2
    - Embedded Assets 2.1.1.1
    - Redactor 2.4.0
    - Retour 3.1.33
    - SEOmatic 3.2.39
    - Sprout Forms 3.7.1.1

**Comment by davidrhoden** [comment]
Deleted it assuming it would appear in that "New widget" dialog.

**Comment by brandonkelly** [comment]
The URL is https://craftcms.com/news.rss.
